packages/@ama-sdk/create/package.json

```diff
@@ -16,6 +16,14 @@
     "postbuild": "cpy package.json dist/",
     "prepare:publish": "prepare-publish ./dist"
   },
+  "peerDependencies": {
+    "@o3r/telemetry": "workspace:^"
```

Comment:
Can this peer dependency ever be provided with npm create?

Reply:
If installed in global only